k8s-deployer/scripts/start-example.sh

```diff
@@ -60,6 +60,7 @@ fi
 if [ "${DO_NOT_CREATE_PROJECT_DIR}" == "" ]; then DO_NOT_CREATE_PROJECT_DIR="false"; fi
 if [ "${DO_NOT_CREATE_PROJECT_DIR}" == "true" ];
 then
+  export DO_NOT_CREATE_PROJECT_DIR
```

Comment:
We do not need to export it. The scope of this variable is only the current script.

Reply:
Will fix it soon ~

Reply:
Issue fixed ~